src/rubrix/client/datasets.py

```diff
@@ -393,6 +403,48 @@ def _from_pandas(cls, dataframe: pd.DataFrame) -> "DatasetForTextClassification"
             [TextClassificationRecord(**row) for row in dataframe.to_dict("records")]
         )
 
+    @_requires_datasets
+    def prepare_for_training(self) -> "datasets.Dataset":
+        """Prepares the dataset for training.
+
+        This will return a "datasets.Dataset" with two columns: "text" and "label"
+        - Records without an annotation are removed.
+        - The "text" column is taken from the _inputs_ attribute of the records.
+            If there is more than one key in the dictionary, the values will be joined together with white space.
```

Comment:
I would discourage this behavior as users would lose the possibility to join the fields however they want (for example adding special tokens) or even just use one of the fields for training. 

My recommendation would be to create one column for each key in the dict

Reply:
:+1: done